applications/Chat/evaluate/eval.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import json
+
+from evaluator import Evaluator
+from utils import jload
+
+def main(args):
+     # load config
+    config=jload(args.config_file)
+
+    if config["language"] == "cn":
+        # get metric settings for all categories
+        metrics_per_category = {}
+        for category in config["category"].keys():
+            metrics_all = {}
+            for metric_type, metrics in config["category"][category].items():
+                metrics_all[metric_type]=metrics
+            metrics_per_category[category]=metrics_all
+
+        answers = {}
+        targets = {}
+
+        with open(args.target_file, 'r', encoding='utf-8') as f1:
+            targets = json.load(f1)
+
+        with open(args.predict_file, 'r', encoding='utf-8') as f2:
+            answers = json.load(f2)
+            
```

Comment:
Use `jload` in `utils.py` instead and remove the `with open` clause.
```python
targets = jload(args.target_file)

answers = jload(args.predict_file)
```

Reply:
okay, I will modify this.